html/06f683b3-c1e3-4e78-8259-e94dc3e3f52b.html

```diff
@@ -174,30 +205,51 @@ <h4>Student Activity </h4>
-    <p><strong>Step 2</strong> - Distribute and simplify.</p>
-    <p>\(3y=x -12\)</p>
-    <p><strong>Step 3 </strong>- Rearrange terms so the \(x\) term and the \(y\) term are both on the left side of the equation.</p>
-    <p>Subtract \(x\) from both sides.</p>
-    <p>\(3y - x = x - x -12\)</p>
-    <p><strong>Step 4</strong> - Simplify.</p>
-    <p>\(3y - x = -12\)</p>
-    <p><strong>Step 5</strong> - Write the equation so the \(x\) term is first.</p>
-    <p>\(-x + 3y = -12\)</p>
-    <p><strong>Step 6</strong> - Multiply each term by -1, so as not to lead with a negative.</p>
-    <p>\(-1(-x + 3y = -12)\)</p>
-    <p>\(x - 3y = 12\)</p>
+    <p><strong>Step 1</strong>- Get rid of the fractions in the equation.<br>
+      Multiply every term by 3.<br>
+      \(3y = 3(\frac{1}{3}x-4)\)</p>
+    <p><strong>Step 2</strong> - Distribute and simplify.<br>
+      \(3y=x -12\)</p>
+    <p><strong>Step 3 </strong>- Rearrange terms so the \(x\) term and the \(y\) term are both on the left side of the equation.<br>
+      Subtract \(x\) from both sides.<br>
+      \(3y - x = x - x -12\)</p>
+    <p><strong>Step 4</strong> - Simplify.<br>
+      \(3y - x = -12\)</p>
+    <p><strong>Step 5</strong> - Write the equation so the \(x\) term is first.<br>
+      \(-x + 3y = -12\)</p>
+    <p><strong>Step 6</strong> - Multiply each term by -1, so as not to lead with a negative.<br>
+      \(-1(-x + 3y = -12)\)<br>
+      \(x - 3y = 12\)</p>
+  </div>
+</div>
+<!--Interaction End --> 
+
+<br>
+<!--Q#-->
+<div class="os-raise-ib-input" data-button-text="Solution" data-fire-event="eventShow" data-schema-version="1.0">
+  <div class="os-raise-ib-input-content">
+    <ol class="os-raise-noindent" start="9">
+      <li>What is the slope of  \(3x + 12y = 24\)? </li>
+    </ol>
+  </div>
+  <div class="os-raise-ib-input-prompt">
+    <p>Enter your answer here:</p>
+  </div>
+  <div class="os-raise-ib-input-ack">
+    <p>Compare your answer:</p>
+    <p>The slope is \(m = -\frac AB\). <br>
+      \(m = -\frac{3}{12}\)<br>
+      \(m = -\frac14\)</p>
   </div>
 </div>
-<!--Interaction End -->
+<!--Interaction End --> 
+<br>
 <br>
 <br>
 <h4>Video: Writing Equations in Different Forms</h4>
 <p>Watch the following video to learn more about writing equations in different forms.</p>
 <div class="os-raise-d-flex-nowrap os-raise-justify-content-center">
-  <div class="os-raise-video-container"><iframe allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen frameborder="0" src="https://www.youtube-nocookie.com/embed/0KhxVDBVGRo?si=31FamA-aJe4EvLpH" title="YouTube video player"></iframe></div>
+    <div class="os-raise-video-container">
+        <iframe src="https://www.youtube.com/embed/0KhxVDBVGRo?si=31FamA-aJe4EvLpH" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen=""></iframe>
```

Comment:
This one just needs the `-nocookie` to be added back so it is: `https://www.youtube-nocookie.com/embed/0KhxVDBVGRo?si=31FamA-aJe4EvLpH`